firebase-installations/gradle.properties

```diff
@@ -1,2 +1,2 @@
-version=16.0.1
+version=16.0.2
 latestReleasedVersion=16.0.0
```

Comment:
I am not about the meaning of these, but does this have to be `v16.0.1`?
```
latestReleasedVersion=16.0.1
```

Reply:
Good catch. I am not sure about the variable either but seems right to update it.